inc/bluetooth/MicroBitBLEManager.h

```diff
@@ -78,6 +79,9 @@ DEALINGS IN THE SOFTWARE.
 #define MICROBIT_BLE_STATUS_STORE_SYSATTR       0x02
 #define MICROBIT_BLE_STATUS_DISCONNECT          0x04
 
+#define MICROBIT_BLE_MODE_PAIRING     0x00
+#define MICROBIT_BLE_MODE_APPLICATION 0x01
+
```

Comment:
@microbit-sam Are these actually used? tracing through, this feels like it might be dead code/causing incorrect state in MicroBitPartialFlashingService line 150?